sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/channel/DistributeInfoTracker.java

```diff
@@ -26,14 +30,25 @@ public DistributeInfoTracker(String distributionGroupId) {
 
     @Override
     synchronized public void onEnqueuingLog(@NonNull Log log, @NonNull String groupName) {
-        if (mDistributionGroupId == null) {
+        if (mDistributionGroupId == null || log instanceof StartServiceLog) {
             return;
         }
 
         /* Set current distribution group ID. */
         log.setDistributionGroupId(mDistributionGroupId);
     }
 
+    @Override
+    public boolean shouldFilter(@NonNull Log log) {
+
+        /* Filter out distribution start session log if session ID is not set (Analytics is disabled). */
+        if (log instanceof DistributionStartSessionLog && log.getSid() == null) {
```

Comment:
This works but will be confusing for aria team as they will also receive this log but will be filtered out without them knowing.

Another solution to it sooner is to use SessionContext from core and check if the last session has a sessionId or null.

Reply:
Implemented using `SessionContext`. Thank you.

Please take a look.